src/main/java/salesforce/lightning/WorkTypeInfo.java

```diff
@@ -0,0 +1,49 @@
+package salesforce.lightning;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import salesforce.BasePage;
+
+public class WorkTypeInfo extends BasePage {
+    @FindBy(xpath = "//*[@class='uiOutputText']")
+    protected WebElement nameWorkTypeTxt;
+
+    @FindBy(xpath = "//*[@class='uiOutputTextArea']")
+    protected WebElement descriptionTxt;
+
+    public WorkTypeInfo(final WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    protected void waitForPageLoaded() {
+        webElementAction.waitForElementVisibility(nameWorkTypeTxt);
+    }
+
+    /**
+     * Gets the name of work type.
+     *
+     * @return name of worktype.
+     */
+    public String getNameOfWorkType() {
+        return nameWorkTypeTxt.getText();
+    }
+
+    public String getDescription() {
+        return descriptionTxt.getText();
+    }
+
+    /**
+     * Sets value and select in txt of Estimated Duration, Block Time Before Appointment, Block Time After Appointment.
+     * Timeframe Start and Timeframe End.
+     *
+     * @param nameTxtBox is name of textBox
+     * @return value of nameTxtBox
+     */
+    public String getTxtField(final String nameTxtBox) {
+        String xpath = String.format("//*[contains(text(),'%s')]/../..//*[@class='uiOutputNumber']", nameTxtBox);
```

Comment:
Same, try to set it as a constant outside the method

Reply:
I use that to generalize and only use that in that method